chain/db.go

```diff
@@ -402,13 +402,14 @@ func (db *DBStore) deleteFileContractExpiration(id types.FileContractID, windowE
 	key := db.encHeight(windowEnd)
 	val := append([]byte(nil), b.getRaw(key)...)
 	for i := 0; i < len(val); i += 32 {
-		if *(*types.FileContractID)(val[i:]) == id {
-			copy(val[i:], val[len(val)-32:])
-			val = val[:len(val)-32]
-			i -= 32
-			b.putRaw(key, val)
-			return
+		if *(*types.FileContractID)(val[i:]) != id {
+			continue
 		}
+		copy(val[i:], val[len(val)-32:])
+		val = val[:len(val)-32]
+		i -= 32
+		b.putRaw(key, val)
+		return
```

Comment:
I don't see how this is an improvement. I understand the general principle of "put the smaller code block inside the conditional," but when the code blocks in question are <5 lines, it becomes more subjective. In this case, we end up with *more* total lines, plus the meaning of the code changes from "look at an element; if it's the one we want, return it" to "look at an element; if it's not the one we want, skip; otherwise, return it" which feels less clear to me.